package.json

```diff
@@ -1,7 +1,4 @@
 {
-  "engines": {
-    "node": "6.6.0"
-  },
```

Comment:
That's probably `curl -sL https://deb.nodesource.com/setup_6.x | bash -` installing 6.11.3-1nodesource1 instead of 6.6.0 ;-)

> `Get:2 https://deb.nodesource.com/node_6.x/ jessie/main nodejs amd64 6.11.3-1nodesource1 [9881 kB]`